assembl/static2/js/app/actions/adminActions/landingPage.js

```diff
@@ -63,23 +63,29 @@ export const updateLandingPageHeaderLogo = (value: File): actionTypes.UpdateLand
 });
 
 export const updateLandingPageModuleTitle = (
-  moduleTypeIdentifier: string,
+  id: string,
   locale: string,
   value: string
 ): actionTypes.UpdateLandingPageModuleTitle => ({
-  moduleTypeIdentifier: moduleTypeIdentifier,
+  id: id,
   locale: locale,
   value: value,
   type: actionTypes.UPDATE_LANDING_PAGE_MODULE_TITLE
 });
 
 export const updateLandingPageModuleSubtitle = (
-  moduleTypeIdentifier: string,
+  id: string,
   locale: string,
   value: string
 ): actionTypes.UpdateLandingPageModuleSubtitle => ({
-  moduleTypeIdentifier: moduleTypeIdentifier,
+  id: id,
   locale: locale,
   value: value,
   type: actionTypes.UPDATE_LANDING_PAGE_MODULE_SUBTITLE
+});
+
+export const createLandingPageModules = (id: string, order: number): actionTypes.CreateLandingPageModules => ({
```

Comment:
should be singular createLandingPageModules => createLandingPageModule

Reply:
Don't you need to have the moduleType identifier as well?

Reply:
typeIdentifier is needed for the CreateLandingPageModule mutation in backend. It's not flagged as required in the backend but really it is.

Reply:
It is required in the frontend though.